### PR TITLE
Made adjustments to the makefiles to support CoCo and CoCo 3 OS-9 in ROM.

### PR DESCRIPTION
**Pull request by boisy**
Thanks to Roger Taylor for these changes.

### CODE REVIEW

**Comment by strickyak** [comment]
What does "in ROM" mean?   Just the Rel/Boot/Krn or all the modules?
Does it imply not having a disk? 
Does it change /DD to /H0, and is that why?

More of these changes seem to be about keydrv_cc3.sb than about ROM.
Is that something obsolete that was still being built?

**Comment by boisy** [comment]
"in ROM" means all modules needed to boot OS-9, including KrnP2, file managers, drivers, etc. Anything that can be crammed into a 32K ROM on a CoCo 3 motherboard (or a 16K ROM on a CoCo 1/2 motherboard).

`sysgo_rom` is a special build of `sysgo` just for this "total ROM" application of OS-9.

`keydrv_cc3.sb` was removed in the CoCo 3 version of OS-9 and merged back into `vtio`. This set of changes just updates the bootlist files for the ROM-based OS-9 build to eliminate that module.

**Review by strickyak — APPROVED** [review]

File: level1/coco1/bootroms/makefile [hunk 1]
```diff
@@ -14,7 +14,7 @@ BOOTFILE_ROM	= $(MD)/rominfo $(MD)/ioman \
 		$(MD)/scf.mn $(MD)/vtio.dr \
 		$(MD)/covdg.io \
 		$(MD)/term_vdg.dt \
-		$(MD)/sysgo_dd \
+		$(MD)/sysgo_rom \
```
[the remark this answers is not in the document]
Comment (boisy) [reply level1/coco1/bootroms/makefile:17]: No. It only changes `sysgo_dd` to `sysgo_rom` for the `BOOTFILE_ROM` macro. That's the boot file specifically for ROM-based installs.

File: level2/coco3/bootlists/dw.bl [hunk 2]
```diff
@@ -121,7 +121,6 @@
 *
 * CoCo 3 I/O sub-drivers
 ../MODULES/SCF/vtio.dr
-../MODULES/SCF/keydrv_cc3.sb
```
[the remark this answers is not in the document]
Comment (boisy) [reply level2/coco3/bootlists/dw.bl:124]: keydrv_cc3.sb is no more -- it's been subsumed back into `vtio` (nee `cc3io`) where it used to be. L. Curtis Boyle did this for EOU, and I carried the change in when I merged EOU changes back into the NitrOS-9 project.

[the remark this answers is not in the document]
Comment (boisy) [reply level2/coco3/bootlists/dw.bl:124]: For context, `keydrv_cc3.sb` was broken out of `cc3io` as part of the Level 2 Upgrade back in 1989, and that carried into the NitrOS-9 project.

**Pull request merged**